deploy_hpo.sh

```diff
@@ -46,6 +47,7 @@ function usage() {
 	echo " -o: build with specific hpo container image name [Default - kruize/hpo:<version>]"
 	echo " -n: Namespace to which hpo is deployed [Default - monitoring namespace for cluster type minikube]"
 	echo " -d: Config maps directory [Default - manifests/configmaps]"
+	echo " Environment Variables to be set: REG_EMAIL, REG_UNAME, REG_PASS"
```

Comment:
I think the variables can be a bit more descriptive, I'd suggest
```
REGISTRY_EMAIL
REGISTRY_USERNAME
REGISTRY_PASSWORD
```

Reply:
Done